### DISCUSSION TITLE
sub grain morphology using NEPER

### DISCUSSION THREAD

**Discussion by neelammishra** [neperfepx/neper#455] [Q&A] [answered]
Hii everyone,
  Is it possible to develop sub-grain morphology in NEPER? For example a microstructure with matrix and precipitate phase.

**Comment by mkasemer** [comment, marked as answer]
Depending on specifics, you can use multi-level tessellations to generate sub-grain features. Very small features relative to the grain size may be difficult to account for.

**Reply by neelammishra** [reply]
is it possible to model precipitates as superellipsoids and matrix as a continuous phase?

**Reply by mkasemer** [reply]
If you mean implement sub-grain ellipsoidal features, I do not believe Neper currently has this capability.

**Reply by neelammishra** [reply]
ok. Thank you for the response

**Reply by neelammishra** [reply]
What does slip parameters signify in FEPX configuration files? 

m 0.010d0 (fixed state strain rate sensitivity)
gammadot_0 1.0d0 (fixed state strain rate  scaling coefficient)
h_0 190.0d0 (strength hardening rate coefficient)
g_0 390.0d0 468.0d0 663.0d0 (initial slip system strength)
g_s0 530.0d0   ?????????
m_prime 1.10d0 (saturation strength rate scaling component)
gammadot_s0 1.0d0  (saturation strength strain rate scaling coefficient)
n 1.0d0 ?????????
what does d0 mean here?? g_s0 ?? n??????

**Reply by mkasemer** [reply]
The above is a question for the FEPX forum, not the Neper forum.

However, the FEPX manual will detail all input parameters and how they relate to the models employed by FEPX. Please consult the FEPX manual first.
